apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -1395,15 +1428,31 @@ private void StartPush()
                 }
             }
 
+            /// <summary>
+            /// This method is called when the acknowledgement request completes.
+            /// This method handles the response from an acknowledgement request for exactly once subscriptions only.
+            /// For non-exactly once subscriptions, <see cref="HandleAckResponse(Task, List{string}, List{string}, List{TimedId})"/> is called.
+            /// </summary>
```

Comment:
Do we expect only one of ackIds, nackIds and extendIds to be non-null? If so, I think that's worth noting.

Reply:
Done.